index.bs

```diff
@@ -931,18 +931,15 @@ When this method is invoked, the user agent MUST execute the following algorithm
             1.  Let |constructCredentialAlg| be an algorithm that takes a [=global object=]
                 |global|, and whose steps are:
 
-                1.  Let |attestationObject| be a new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the
-                    bytes of <code>|credentialCreationData|.[=attestationObjectResult=]</code>'s value.
-
                 1.  If <code>|credentialCreationData|.[=attestationConveyancePreferenceOption=]</code>'s value is
                     <dl class="switch">
                         :   "none"
-                        ::  Replace potentially uniquely identifying information (such as [=AAGUID=] and
-                            [=attestation certificates=]) in the [=attested credential data=] and [=attestation statement=],
-                            respectively, with blinded versions of the same data.
-
-                        Issue(w3c/webauthn#694): need to define "blinding".  See also
-                            [#462](https://github.com/w3c/webauthn/issues/462).
+                        ::  Replace potentially uniquely identifying information with non-identifying versions of the
+                            same:
+                               1. If the [=AAGUID=] in the [=attested credential data=] is 16 zero bytes, <code>|credentialCreationData|.[=attestationObjectResult=].fmt</code> is "packed", and "x5c" & "ecdaaKeyId" are both absent from <code>|credentialCreationData|.[=attestationObjectResult=]</code>, then [=self attestation=] is being used no further action is needed.
```

Comment:
Missing punctuation just before "no further action" near the end of this line.

Reply:
Done.